docs/fr-FR/config/README.md

```diff
@@ -1100,6 +1101,36 @@ The `ocaml` module shows the currently installed version of OCaml. Le module est
 symbol = "🐪 "
 ```
 
+## Perl
```

Comment:
The translations are automatically updated by crowdin. So please revert all the changes to the lang files.